### DISCUSSION TITLE
Pip install failing on python 3.10.11 and 3.10.13

### DISCUSSION THREAD

**Discussion by wyzaguirre** [Ulm-IQO/qudi-core#85] [Q&A] [answered]
Hi there. I'm trying to get qudi-core installed, but I'm getting:
ERROR: Could not find a version that satisfies the requirement PySide2==5.15.2.1 (from qudi-core) (from versions: none)
ERROR: No matching distribution found for PySide2==5.15.2.1
When running: 
python -m pip install -e .
Do I need a specific 3.10 version?

**Comment by wyzaguirre** [comment, marked as answer]
Ah, nevermind. It seems that PySide2 is incompatible with the new apple M chips. Have to make a special x86 virtual env to work

**Reply by Neverhorst** [reply]
You are right. 
Apple Silicon is [supported from version `Qt 6.2+`](https://www.qt.io/blog/qt-on-apple-silicon) and `Qt for Python` [supports it from version `PySide6 6.2.2+`](https://wiki.qt.io/Qt_for_Python) (See also [here](https://forum.qt.io/topic/124289/mac-m1-pyside-2-6-can-t-install-lib/5)).

So until we transition `qudi` to `PySide6`, I'm afraid it has to run in a `x86` emulation environment. This is on the roadmap but probably not going to happen within the next couple of months.

**Reply by wyzaguirre** [reply]
I got it working using the built in rosetta 2, would you like me to add the method to the docs?

**Reply by Neverhorst** [reply]
If you could add it [here](https://github.com/Ulm-IQO/qudi-core/blob/main/docs/setup/installation.md) and open a PR, that would be greatly appreciated. 👍🏻

**Comment by timoML** [comment]
If you managed to get qudi running on a recent MacOS, I think documenting it here (or somewhere in our documentation) would be very valueable to others.
